examples/existing-instance/outputs.tf

```diff
@@ -0,0 +1,38 @@
+########################################################################################################################
+# Outputs
+########################################################################################################################
+
+output "crn" {
+  description = "CRN of the existing watsonx Governance instance."
+  value       = module.existing_watsonx_governance_instance.crn
+}
+
+output "account_id" {
+  description = "Account ID of the existing watsonx Governance instance"
+  value       = module.existing_watsonx_governance_instance.account_id
+}
+
+output "id" {
+  description = "ID of the watsonx Governance instance."
+  value       = module.existing_watsonx_governance_instance.id
+}
+
+output "name" {
+  description = "Name of the existing watsonx Governance instance"
+  value       = module.existing_watsonx_governance_instance.name
+}
+
+output "guid" {
+  description = "GUID of the existing watsonx Governance instance"
+  value       = module.existing_watsonx_governance_instance.guid
+}
+
+output "plan_id" {
```

Comment:
I don't think we will have `plan_id` or `dashboard_url` in case of existing instance, so these two can be omitted from outputs. Refer local validation [here](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-governance/blob/c9509a79184afcdb849e63d96d8a1e52267b5d36/main.tf#L10) and [here](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-governance/blob/c9509a79184afcdb849e63d96d8a1e52267b5d36/main.tf#L11)